### ISSUE TITLE
Explain offline mode and PWA mode

### ISSUE THREAD

**Issue by wavebeem** [wavebeem/pkmn.help#147]
I bet a lot of people don't understand offline mode for websites, or adding the app to their home screen. Maybe some text would help.

**Comment by FruityGumGuiltyFruit** [comment]
Is there a method that allows one to use this tool offline? I tried using save page, but that didn't work. Unfortunately, I haven't the slightest idea how feasible it would be to add support for offline use or making this a downloadable program.

**Comment by wavebeem** [comment]
Yup! This guide covers how to install a PWA (progressive web app)

https://mobilesyrup.com/2020/05/24/how-install-progressive-web-app-pwa-android-ios-pc-mac/

If you do this, the site should function offline. The only exception is that Pokédex icons are not stored locally on your device. But the rest of the app should work just fine.

**Comment by wavebeem** [comment]
Actually, that was just how to install it as an "app" on your device. If you just want to use it offline you can simply go to `pkmn.help` in your browser and it will work automatically if you've used it before while online. Test it out with Airplane Mode on your phone or something.

**Comment by FruityGumGuiltyFruit** [comment]
That works great, thank you. :)
